hoa-backend/src/auth/auth.controller.ts

```diff
@@ -1,19 +1,28 @@
 import { Body, Controller, Post } from '@nestjs/common';
 import { LogInDTO } from './dto/login.dto';
 import { AuthService } from './auth.service';
+import { CustomExceptionFilter } from './dto/custom-exception.filter';
+import { UseFilters } from '@nestjs/common';
 import { SignUpDTO } from './dto/signup.dto';
 
 @Controller('auth')
 export class AuthController {
   constructor(private readonly authservice: AuthService) {}
 
+  @UseFilters(new CustomExceptionFilter())
```

Comment:
since you added `CustomExceptionFilter` as a global filter, is it still needed here?

Reply:
hmmm can you tell me how did I make this Global?